lib/src/value_layout_builder.dart

```diff
@@ -58,10 +57,9 @@ class ValueLayoutBuilder<T>
   ///
   /// The [builder] argument must not be null.
```

Comment:
We can also remove this one

Reply:
@John-Dormevil This would still need to be changed. After that, all suggested changes should have been made.

@letsar Can we merge this after this change has been applied? Then, I will change https://github.com/letsar/flutter_sticky_header/pull/58 accordingly and it will be ready for review/merge/publish as well :)